Makefile

```diff
@@ -57,3 +57,6 @@ collectstatic:
 
 lint: 
 	poetry install --sync && poetry run pre-commit run --all-files
+
+css:
+	poetry run ./manage.py custom_bootstrap5
```

Comment:
Do you think this should always run at the start of the `collectstatic` make command (above), since it generates a file that gets stored in `web/static`? If so, can I add `make css` or the corresponding line to the start of `collectstatic`?

Reply:
We answered this question in another conversation.  We've decided to commit the custom css file to our repo, so there is no need to worry about the timing of these commands.